charts/zitadel/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: zitadel
 description: A Helm chart for ZITADEL v2
 type: application
-appVersion: "v2.21.1"
+appVersion: "v2.22.0"
 version: 4.2.0
```

Sorry, suggested it wrong before :slightly_smiling_face: 

```suggestion
appVersion: "v2.23.1"
version: 4.3.0
```